addons/im_livechat/models/chatbot_script_step.py

```diff
@@ -342,11 +342,6 @@ def _process_step_forward_operator(self, discuss_channel):
         # handle edge case where we found yourself as available operator -> don't do anything
         # it will act as if no-one is available (which is fine)
         if human_operator and human_operator != self.env.user:
-            discuss_channel.sudo().add_members(
-                human_operator.partner_id.ids,
-                open_chat_window=True,
-                post_joined_message=False)
-
             # rename the channel to include the operator's name
```

Comment:
I'd rather renaming the channel after adding the operator to the channel, it would be easier to follow this flow:
- finding an operator
- adding him to the channel
- renaming the channel